modules/beopBidAdapter.md

```diff
@@ -0,0 +1,50 @@
+# Overview
+
+**Module Name** : BeOp Bidder Adapter  
+**Module Type** : Bidder Adapter  
+**Maintainer** : tech@beop.io
+
+# Description
+
+Module that connects to BeOp's demand sources
+
+# Test Parameters
```

Comment:
Can these parameters be used to get an ad bid? I tried putting the 1st ad unit in a test page and failed to get a response from your server due to a CORS error:
```
Access to XMLHttpRequest at 'https://s.beop.io/bid' from origin 'http://localhost:3000' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
```

Reply:
Hello, we are updating our server config to manage CORS asap. Will re-ask for the review once it is ok

Reply:
BTW, the accountId param is not an actual one, so you can't really have an ad. The publisher will need to create an account on BeOp Platform before using our Prebid adapter

Reply:
In that case, how can this adapter be tested from a test page? With an invalid account id, what will the server response be?

Reply:
With an invalid account id, it will answer a HTTP 400 error, as this is a bad request. We will provide a valid account id for tests purposes

Reply:
Update has been done concerning the CORS server management + the valid account ID for testing 🙏

Reply:
I'm still seeing the same response.
```
Access to XMLHttpRequest at 'https://hb.beop.io/bid' from origin 'http://localhost:3000' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
```

Reply:
Hi! Sorry about the cors error. Should be good now.

Reply:
CORS issue is resolved, but the request to https://hb.beop.io/bid now returns `400 No Content`.
Here is the request POST body:
```
{"at":"Wed Aug 11 2021 10:29:04 GMT-0400 (Eastern Daylight Time)","pid":"5a8af500c9e77c00017e4cad","url":"http://localhost:3000/","lang":"en-US","kwds":[],"dbg":false,"slts":[{"sizes":[[300,250]],"pid":"5a8af500c9e77c00017e4cad","bid":"27a5f1422f3156","brid":"1d3aae94c6133c","name":"div-gpt-ad-1460505748561-0","aid":"fbb5466e-4440-413d-a685-7c1b9458c29c","tid":"fe9a2b17-76a7-4772-8ba3-6f07f9af0064","brc":1,"bdrc":"","bwc":""}],"tc_string":""}
```

Reply:
You're right. Should be good now.

Reply:
I tried again. Now, no bids are returned. The test params should return some sample ad, so the adapter can be fully tested using a test page.

Reply:
@msm0504 We have a working example with the following page: https://gist.github.com/bloodyowl/1d51fe7f550f97dbb317da3916c1cc16